### ISSUE TITLE
`lowerFirst` filter not working

### ISSUE THREAD

**Issue by Usipov** [krzysztofzablocki/Sourcery#316]
Hello! Thanks for the amazing tool. Found an issue with `lowerFirst` filter.

[lowerFirstNotWorking.zip](https://github.com/krzysztofzablocki/Sourcery/files/1021257/lowerFirstNotWorking.zip)

Attached a zip with a structure
`- generated`: output files
`- scripts`: one using a template with `lowerFirst`, other - with `lowerFirstWord` and last one - with `upperFirst`
`- Sourcery-0.6` - the binaries taken from github
`- sources` - files to be compiled
`- templates` - templates for the scrips. one with `lowerFirst` filter, other - with `lowerFirstWord` filter and last one - with `upperFirst` filter

Generally the templates look like that:
```
// sourcery:file:generated/<#fiilterName#>.swift
func foo() {
    {% for type in types.implementing.Protocol %}
    let {{ type.name|<#fiilterName#> }}
    {% endfor %}   
}
// sourcery:end
```

And the scrips look like that:
```
#!bin/bash

SCRIPT_ROOT=`cd "$(dirname "$0")" && pwd`
ROOT="$SCRIPT_ROOT/.."
"$ROOT/Sourcery-0.6/bin/sourcery" --sources "$ROOT/sources" --templates "$ROOT/templates/<#fiilterName#>.stencil"
```

`lowerFirst.sh` logs the following:
```
# Scanning sources...
# Found 4 types.
# Loading templates...
# Loaded 1 templates.
# Generating code...
# Unknown filter 'lowerFirst'
```
And according to [this](https://github.com/krzysztofzablocki/Sourcery/blob/4e920792664029203820fba1a63ba4ba925f552e/SourceryTests/Generating/StencilTemplateSpec.swift) test, `lowerFirst` is expected to work 

Other scripts work as expected. 
I used `lowerFirstWord` for my needs as a workaround, but it is a kludge anyway =)
Thanks in advance

**Comment by Antondomashnev** [comment]
Hi, @Usipov thanks for reporting it 👍 . I'll try to take a look in it today.

**Comment by ilyapuchka** [comment]
Please see [CHANGELOG](https://github.com/krzysztofzablocki/Sourcery/blob/master/CHANGELOG.md). `lowerFirst` is implemented only on master, it was not shipped with 0.6 version.

**Comment by dtartaglia** [comment]
I think this needs to be reopened. I'm using the cocoapod v0.7.2.

    {{ variable.typeName | lowerFirst }}

Does not produce a lowercased typename? (I just started using the tool, maybe I'm doing something wrong.)
